### ISSUE TITLE
Pierwsza wersja prototypowej aplikacji Android

### ISSUE THREAD

**Issue by qaziok** [cvlab-ai/polish_banknotes#26]
Fajnie, aby pojawił się już pierwszy prototyp aplikacji na androida

**Comment by jankejc** [comment]
"Prototyp aplikacji" jest naszym celem na ten semestr, więc czy moglibyśmy zmienić tytuł issue na np. "Pierwszy prototyp aplikacji"? Dzięki temu moglibyśmy iterować te nazwy issue dla kolejnych prototypów, a i w historii issue będzie to lepiej wyglądało myślę.

**Comment by jankejc** [comment]
Udane uruchomienie przykładowej aplikacji wykorzystującej model TensorFlow Lite. Zamieszczam prezentacje. Do stworzenia branch'a czekam aż @kjedrasz2137 podeśle mi wstępny model sieci.

https://user-images.githubusercontent.com/26117679/202489294-4169e091-bf9e-412d-b25e-c902b6f6fe40.mp4

**Comment by jankejc** [comment]
Ponowne uruchomienie przykładowej aplikacji wykorzystującej model TensorFlow Lite z **lepszego** [tutoriala](https://github.com/tensorflow/examples/tree/master/lite/examples/image_classification/android) podlinkowanego przez @fzemi. Zamieszczam prezentacje.

https://user-images.githubusercontent.com/26117679/203008385-8b23f01d-2f28-4e2c-833e-1062af42d0db.mp4

**Comment by jankejc** [comment]
Przy aktywnej współpracy z @kjedrasz2137 udało się odpalić pierwszy prototyp aplikacji z naszym modelem. Kluczem było dodanie metadanych bez normalizacji w skrypcie i softmax przy generowaniu modelu. Model był już znormalizowany podczas generowania.

https://user-images.githubusercontent.com/26117679/204250476-9da03d35-cac4-4d3f-981e-76f6d839693a.mp4
